feature/networkinstance/otg_tests/defaults_test/defaults_test.go

```diff
@@ -103,6 +104,7 @@ var (
 		IPv6Len: 64,
 		MAC:     "02:00:02:01:01:01",
 	}
+	kneDeviceModelList = []string{"ncptx", "ceos", "ixr10e", "8201", "xrd"}
 )
```

Comment:
can we add srlinux instead of 'ixr10e' and '8201'

Reply:
updated , please check.